harmony-api/prisma/schema.prisma

```diff
@@ -0,0 +1,168 @@
+datasource db {
+  provider = "postgresql"
+  url      = env("POSTGRES_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  user_id    Int       @id @default(autoincrement())
+  username   String    @unique
+  password   String
+  created_on DateTime  @default(now())
+  active     Boolean?
+  posts      Post[]
+  likes      Like[]
+  comments   Comment[]
+  Artist     Artist[]
+  Follows    Follow[]  @relation("Following")
+  Followers  Follow[]  @relation("Follower")
+}
+
+model Artist {
+  artist_id    Int            @id @map("user_id")
+  artist_name  String         @unique
+  artist_alias String?
+  user         User           @relation(fields: [artist_id], references: [user_id])
+  albums       Album[]
+  songs        Song[]
+  Publish      Publish[]
+  RelateArtist RelateArtist[]
+}
+
+model Post {
+  post_id           Int            @id @default(autoincrement())
+  user_id           Int
+  title             String
+  post_time         DateTime       @default(now())
```

Comment:
I believe using createdAt is a pretty standard name for this type of thing, at least in Prisma, it will probably make it easier for us to program if we use it for consistency 
i.e.
```
model Post {
  ...
  createdAt DateTime @map("created_at") @default(now())
  ...
```
And the same goes for the similar field in `User` and `Follow`

Reply:
I totally agree with that. See (f15a2d6)